content/en/shift_left.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Shift Left
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. Although originally used to refer to the process of testing early, Shift Left can now also be applied to other aspects of software development and DevOps, such as Security and Deployment. 
+
+## Problem it addresses
+Security issues, bugs, and software defects can be more difficult and expensive to fix if they are discovered late in the development cycle or after deployment, particuarly if the software has already been deployed into production. 
+
+## How it helps
+By adopting a shift left mindset when it comes to software development, teams can implement testing and security throughout the development lifecycle. By sharing responsibility for testing and security, the development team--from   software engineers to quality assurance to devops/operations--can all play a part in ensuring the stability and security of an application. 
+
+In addition, shifting left enables continuous improvement and follows more of an agile rather than a waterfall approach to development, as teams can make small iterative improvements and identify problems earlier on. Security and secure development practices can be adopted as early as the design and architecture phase, rather than as an afterthought. The time required for testing before a software release can also be much faster if testing has already been done throughout the development cycle. 
```

Comment:
I tried to simplify two sentences in this paragraph. Let me know if you're ok with it. 

The afterthought comment feels a bit judgmental.  They may not have the ability to do it earlier. Maybe we should stop after "as soon as the design and architecture phase"